src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -972,6 +1003,51 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 		return true;
 	}
 
+	/**
+	 * Refresh Azure access token
+	 * @param uri connection uri
+	 * @returns true if no need to refresh or successfully refreshed token
+	 */
+	public async refreshAzureAccountTokens(profile: ConnectionProfile): Promise<boolean> {
```

Comment:
This seems to be very similar to the logic in refreshAzureAccountTokenIfNecessary - can you have that call this instead?